src/package.json

```diff
@@ -69,12 +68,12 @@
     }
   },
   "peerDependencies": {
-    "@angular/animations": ">=15.0.0 <17.0.0",
-    "@angular/cdk": ">=15.0.0 <17.0.0",
-    "@angular/common": ">=15.0.0 <17.0.0",
-    "@angular/compiler": ">=15.0.0 <17.0.0",
-    "@angular/core": ">=15.0.0 <17.0.0",
-    "@angular/forms": ">=15.0.0 <17.0.0",
+    "@angular/animations": ">=16.0.0 <18.0.0",
```

Comment:
The codesandbox dependencies I think need updated. Looks like it is still using Angular 15:

![image](https://github.com/UXAspects/UXAspects/assets/20795331/bdf967d8-ba0c-489b-8408-f0f59c3b8afd)